### PR TITLE
Fixing v1-alpha tests

### PR DESCRIPTION
**Pull request by saxenism**
Fixing all existing Pooled Credit Line tests from v1-alpha.

### CODE REVIEW

File: contracts/mocks/MockVerification2.sol [hunk 1]
```diff
@@ -4,9 +4,21 @@ pragma solidity 0.7.6;
 contract MockVerification2 {
 
     mapping (address => bool) public verifiers;
+    mapping (address => bool) public verifiedUsers;
 
-    function isUser(address, address) public pure returns (bool) {
-        return true;
+    function isUser(address _user, address _verifier) public view returns (bool) {
+        if(!verifiers[_verifier]) {
```
Comment (prateekreddy) [inline contracts/mocks/MockVerification2.sol:10]: return verifiers[_verifier] will do na

File: contracts/mocks/MockVerification2.sol [hunk 2]
```diff
@@ -4,9 +4,21 @@ pragma solidity 0.7.6;
...
+            return false;
+        }
+        return verifiedUsers[_user];
+    }
+
+    function verifyUser(address _user, address _verifier) public {
+        if(verifiers[_verifier]) {
+            verifiedUsers[_user] = true;
```
Comment (prateekreddy) [inline contracts/mocks/MockVerification2.sol:18]: let's add seperate function if you wanna remove verified user

File: contracts/test/PooledCreditLines/PooledCL_Liquidations.t.sol [hunk 3]
```diff
@@ -1,1016 +1,1032 @@
...
+        verification.verifyUser(address(pooledCreditLineLender_2), address(_lenderVerifier));
+        verification.verifyUser(address(pooledCreditLineLender_3), address(_lenderVerifier));
+    }
+
+/*****************************************************************************************************************************************************
+    Expected Behavior: Pass
+********************************************************************************************************************************************************/
+
+    // // Test5: Lender_1 liquidates the credit lines, withdraws his liquidity and other lenders get their correct share of liquidated collateral
+    // function test_pooledCreditLine_LiquidationTests_LiquidationDueToLowCollateralRatio_MultipleLenders_LiquidityWithdrawn(
```
Comment (prateekreddy) [inline contracts/test/PooledCreditLines/PooledCL_Liquidations.t.sol:235]: why is this commented out ?

**Review by prateekreddy — CHANGES_REQUESTED** [review]

File: contracts/mocks/MockVerification2.sol (hunk 1 above)
Comment (saxenism) [reply contracts/mocks/MockVerification2.sol]: True. Changed the mapping structure. Please review again.

File: contracts/test/PooledCreditLines/PooledCL_Liquidations.t.sol (hunk 3 above)
Comment (saxenism) [reply contracts/test/PooledCreditLines/PooledCL_Liquidations.t.sol]: As discussed, it was failing. With relevant changes in the test, the test now works.

Can also add the edge case detected in the contract side.
